packages/ui/src/color-picker/ColorPicker.tsx

```diff
@@ -0,0 +1,221 @@
+import { ColorPicker as ArkColorPicker, Portal } from "@ark-ui/react";
+import { EyeDropperIcon } from "@heroicons/react/24/outline";
+import { cva } from "class-variance-authority";
+import {
+  type ComponentPropsWithoutRef,
+  type ElementRef,
+  forwardRef,
+} from "react";
+import { Button } from "../button";
+import { InputField } from "../input-field";
+import type { ValueOrFunction } from "../types";
+import { getValue } from "../utils";
+
+const COLORS = [
+  "red",
+  "blue",
+  "green",
+  "yellow",
+  "pink",
+  "black",
+  "brown",
+  "orange",
+  "indigo",
+  "maroon",
+  "purple",
+];
+
+const colorPickerTriggerClasses = {
+  sm: "size-[30px] min-w-[30px] min-h-[30px] p-1",
+  md: "size-[38px] min-w-[38px] min-h-[38px] p-1.5",
+  lg: "size-[46px] min-w-[46px] min-h-[46px] p-2",
+};
+
+const colorPickerContentClasses = cva(
+  "dark:bg-secondary-900 border-secondary-200 dark:border-secondary-600 rounded-lg border bg-white drop-shadow-lg w-full",
+  {
+    variants: {
+      size: {
+        sm: "md:max-w-[280px] max-w-[250px] p-3 space-y-3",
+        md: "md:max-w-[330px] max-w-[290px] p-4 space-y-4",
+        lg: "md:max-w-[380px] max-w-[310px] p-5 space-y-[18px]",
+      },
+    },
+  },
+);
+
+const colorPickerAreaClasses = cva("overflow-hidden rounded-md", {
+  variants: {
+    size: {
+      sm: "h-[120px]",
+      md: "h-[140px]",
+      lg: "h-[160px]",
+    },
+  },
+});
+
+const colorPickerSwatchClasses = cva("rounded-md", {
```

Comment:
Change the border radius according to size and should be one less then the popover content's border radius